DEVNOTE.md

```diff
@@ -45,3 +47,37 @@ To setup it:
 4. In *Credentials* section, set up an `ssh` connection to the UTBot docker container you have. Consider enabling connection via OpehSSH authentification agent to ease files synchronization.
 5. All other fields should fill in automatically.
 
+
+## Troubleshooting the build
+### read -i invalid option
+You can experience this on Mac OS:
+```shell
+read: -i: invalid option
+```
+It is because Mac OS has too old bash version. Upgrade bash version and set it to default:
+```shell
+brew install bash
+chsh -s /usr/local/bin/bash
+```
+### Problem with gRPC
+If you experience this problem:
+```shell
+--grpc_out: protoc-gen-grpc: Plugin failed with status code 1.
+/bin/grpc_cpp_plugin: program not found or is not executable
+```
+```shell
+ninja: error: 'protobuf/testgen.grpc.pb.h', needed by 'CMakeFiles/utbot.dir/main.cpp.o', missing and no known rule to make it
+```
+Copy the program to /bin:
+```shell
+cp /utbot_distr/install/bin/grpc_cpp_plugin /bin/
+```
+### Error while mounting empty volume
+If this error happens on Mac OS:
+```shell
+docker: Error response from daemon: error while mounting volume '': VolumeDriver.Mount: exit status 1%!(EXTRA []interface {}=[]).
```

Comment:
Similar problem fixed in another plugin https://github.com/trajano/docker-volume-plugins/issues/29